modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -580,6 +585,34 @@ public static Preferences getPreferences() {
      */
     public interface Preferences extends ObservableMap<String, Object> {
 
+        /**
+         * Specifies whether applications should minimize the amount of non-essential animations,
+         * reducing discomfort for users who experience motion sickness or vertigo.
+         * <p>
+         * If the platform does not report this preference, this property defaults to {@code false}.
+         *
+         * @return the {@code reducedMotion} property
+         * @defaultValue {@code false}
+         * @since 24
+         */
+        ReadOnlyBooleanProperty reducedMotionProperty();
```

Comment:
Adding a new method  without a default implementation to an interface is an incompatible change. Since we do not expect that an application would ever implement this interface, this seems OK to me.

We should either add an `@implSpec` to the class docs indicating that applications should not implement  `Platform.Preferences` or make it a sealed interface.

Reply:
Good point, I've sealed the interface.